faker.go

```diff
@@ -113,6 +113,9 @@ const (
 	BoundaryEnd           = "boundary_end"
 	Equals                = "="
 	comma                 = ","
+	colon                 = ":"
```

Comment:
Are there any reasons why we choose colon for separator?

From the issue, he expecting some kind of ",", I'm okay with either, but it will be great if we also considering the user experience? 

How usually people separate a list of items? 🤔

Reply:
yeah I thought about that. I can switch it if you want. I basically was lazy and didn't want to do the work to make `oneof: value1, value2, value3`. I didn't have an actually good reason for that.

Reply:
I have to run an errand for a little bit right now. But when I get back in an hour or 2, I will be happy to make the user experience better.
i.e. support instead `oneof: value1, value2, value` <- use colon for the `oneof` tag only. Then use commas to actually separate the "arguments" to the tag.

Reply:
@bxcodec I think I have it in place now. It now uses `,` to separate the "arguments" to the `oneof` tag

Reply:
Nice, thanks bro @andrew-werdna